### PR TITLE
feat: added Bridge Bsc operator balance checker

### PR DESCRIPTION
**Pull request by shkangr**
Waiting for the value of the threshold to be set

### CODE REVIEW

File: bridge-status-bot/app.sh [hunk 1]
```diff
@@ -44,6 +45,8 @@ ETH_BALANCE=$(get_eth_balance "$BRIDGE_ETHEREUM_ADDRESS")
 ETH_BALANCE_THRESHOLD=0.5
 ETH_SAFE_EXECUTOR_BALANCE=$(get_eth_balance "$BRIDGE_ETHEREUM_SAFE_EXECUTOR_ADDRESS")
 ETH_SAFE_EXECUTOR_THRESHOLD=2
+ETH_BRIDGE_BSC_OPERATOR_BALANCE=$(get_eth_balance "$BRIDGE_BSC_OPERATOR_ADDRESS")
+ETH_BRIDGE_BSC_OPERATOR_THRESHOLE=2
```
Comment (moreal) [inline bridge-status-bot/app.sh:49]: Maybe it seems like typo

```suggestion
ETH_BRIDGE_BSC_OPERATOR_THRESHOLD=2
```

Comment (ahramyun7) [inline bridge-status-bot/app.sh:49]: 오타 수정 부탁드립니다.
`ETH_BRIDGE_BSC_OPERATOR_THRESHOLE` -> `ETH_BRIDGE_BSC_OPERATOR_THRESHOLD`

File: bridge-status-bot/app.sh [hunk 2]
```diff
@@ -63,7 +66,8 @@ TEXT=":notebook: *9c-bridge report*\\n\
 :ncg: NCG Balance:                                           *$GOLD_BALANCE*\\n\
 GAP between :wncg: and :ncg::                           *$GAP*
 :ether: Ether Safe Contract balance:                  *$ETH_BALANCE*\\n
-:ether: Ether Safe Contract Executor balance: *$ETH_SAFE_EXECUTOR_BALANCE*\\n"
+:ether: Ether Safe Contract Executor balance: *$ETH_SAFE_EXECUTOR_BALANCE*\\n
+:ether: Ether Bridge Bsc Operator balance: *$ETH_BRIDGE_BSC_OPERATOR_BALANCE*\\n"
```
Comment (ahramyun7) [inline bridge-status-bot/app.sh:70]: `Bsc -> BSC`

File: bridge-status-bot/app.sh (hunk 1 above)
Comment (shkangr) [reply bridge-status-bot/app.sh]: Fixed!

Comment (shkangr) [reply bridge-status-bot/app.sh]: 수정했습니다 😀

File: bridge-status-bot/app.sh (hunk 2 above)
Comment (shkangr) [reply bridge-status-bot/app.sh]: Fixed!

**Review by moreal — APPROVED** [review]

**Pull request merged**